integration-tests/dynamic-config-tests/src/test/java/org/apache/dynamicconfig/test/DynamicConfigurationIT.java

```diff
@@ -0,0 +1,144 @@
+    Log4jUtils.init();
+    BeanUtils.init();
+    url = DynamicPropertyFactory.getInstance().getStringProperty("apollo.config.serverUri", "missing").getValue();
+    token = DynamicPropertyFactory.getInstance().getStringProperty("apollo.config.token", "missing").getValue();
+  }
+
+  @After
+  public void tearDown() throws Exception {
+    //delete
+    clearConfiguration();
+  }
+
+  //delete configuration items set by test code
+  public void clearConfiguration() {
+    String delete = url + "/openapi/v1/envs/DEV/apps/SampleApp/clusters/default/namespaces/application/items/loadbalance?operator=apollo";
+    String release = url + "/openapi/v1/envs/DEV/apps/SampleApp/clusters/default/namespaces/application/releases";
+    RestTemplate rest = new RestTemplate();
+    HttpHeaders headers = new HttpHeaders();
+    headers.add("Content-Type", "application/json;charset=UTF-8");
+    headers.add("Authorization", token);
+    HttpEntity<?> entity = new HttpEntity<Object>(headers);
+
+    ResponseEntity<String> exchange = rest.exchange(delete, HttpMethod.DELETE, entity, String.class);
+    Map<String,String> body = new HashMap<>();
+    body.put("releaseTitle", "release-configuration");
+    body.put("releasedBy", "apollo");
+    entity = new HttpEntity<Object>(body, headers);
+    exchange = rest.exchange(release, HttpMethod.POST, entity, String.class);
+
+  }
+
+  @Test
+  public void testDynamicConfiguration() {
+    //before
+    Assert.assertEquals(DynamicPropertyFactory.getInstance().getStringProperty("loadbalcance", "default").getValue(), "default");
+
+    //set and return 200. release 200,update return 200
+
+    String setLoadBalance = url + "/openapi/v1/envs/DEV/apps/SampleApp/clusters/default/namespaces/application/items";
+
+    String release = url + "/openapi/v1/envs/DEV/apps/SampleApp/clusters/default/namespaces/application/releases";
+    RestTemplate rest = new RestTemplate();
+    HttpHeaders headers = new HttpHeaders();
+    headers.add("Content-Type", "application/json;charset=UTF-8");
+    headers.add("Authorization", token);
+
+    //set new configuration item
+    Map<String, String> body = new HashMap<>();
+    body.put("key", "loadbalance");
+    body.put("value", "roundrobbin");
+    body.put("dataChangeCreatedBy", "apollo");
+    HttpEntity<?> entity = new HttpEntity<Object>(body, headers);
+
+    ResponseEntity<String> exchange = rest.exchange(setLoadBalance, HttpMethod.POST, entity, String.class);
+    Assert.assertEquals(exchange.getStatusCodeValue(), HttpStatus.OK.value());
+
+    //relese loadbalance item
+    body.clear();
+    body.put("releaseTitle", "release-configuration");
+    body.put("releasedBy", "apollo");
+    entity = new HttpEntity<Object>(body, headers);
+    exchange = rest.exchange(release, HttpMethod.POST, entity, String.class);
+    Assert.assertEquals(exchange.getStatusCodeValue(), HttpStatus.OK.value());
+
+    //waiting for a refresh cycle
+    await().atMost(5, SECONDS).until(
```

Comment:
as you set the refresh interval to be 2 seconds, wait for 3 seconds is enough.

Reply:
I guess atMost will finish the check before it really takes 5 seconds.